From: Ansgar Burchardt
Date: Sat, 17 Jan 2009 18:34:59 +0100
Subject: Use ~/.simutrans for savegames

This patch reverts SVN commit 1937 to make simutrans use ~/.simutrans for
savegames instead of ~/simutrans.

--- simutrans.orig/simsys_s.cc
+++ simutrans/simsys_s.cc
@@ -317,7 +317,7 @@
 	return NULL;
 #else
 #ifndef __APPLE__
-	sprintf( buffer, "%s/simutrans", getenv("HOME") );
+	sprintf( buffer, "%s/.simutrans", getenv("HOME") );
 #else
 	sprintf( buffer, "%s/Library/Simutrans", getenv("HOME") );
 #endif
